src/main/java/com/minecolonies/structures/helpers/Structure.java

```diff
@@ -185,11 +189,17 @@ public static File getClientSchematicsFolder()
      * - cache
      * - schematics folder
      * - jar
-     * It should be the exact oppsite that the way used to buikd the list.
+     * It should be the exact opposite that the way used to build the list.
+     *
+     * Suppressing Sonar Rule squid:S2095
+     * This rule enforces "Close this InputStream"
+     * But in this case the rule does not apply because
+     * We are returning the stream and that is reasonable
      *
      * @param structureName name of the structure to load
      * @return the input stream or null
      */
+    @SuppressWarnings("squid:S2095")
```

Comment:
because not all random suppressions are allowed so I deactivate it manually each time